website/content/docs/job-specification/multiregion.mdx

```diff
@@ -67,6 +67,15 @@ a time. When each region completes its local deployment, it enters a `blocked`
 state where it waits until the last region has completed the deployment. The
 final region will unblock the regions to mark them as `successful`.
 
+## Multiregion Parameterized Job Dispatch
+
+Job dispatching is region specific. While a [parameterized job] can be 
+deployed to multiple [federated regions] like any other job, a parameterized
+job operates much like a function definition that takes variable input.
+Operators are expected to invoke the job by invoking [`job dispatch`] 
+from the CLI or the [HTTP API] and provide the appropriate dispatch options
+for that region.
+
```

Comment:
It may be worth adding a note about periodic jobs here as well. They will run at the same "global" time right? Like 5PM UTC, or whatever timezone you set in https://developer.hashicorp.com/nomad/docs/job-specification/periodic#time_zone